src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/index.js

```diff
@@ -430,6 +430,10 @@ Component.register('sw-product-detail', {
         },
 
         onSave() {
+            if (!this.validateProductPurchase()) {
+                return;
```

Comment:
Does the user get a notification if the max purchase quantity exceeds the min purchase quantity?

Reply:
Ahh, now I see where this happens. I would move the `createNotificationError` in the `onSave` method. Because as a developer I would not expect a notification warning when I use a validate function :)

Reply:
Yes that is true, changed it.